libs/src/utils/fileHasher.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from 'ipfs-unixfs-importer'
 import fs from 'fs'
 import { hrtime } from 'process'
-import { promisify } from 'util'
+import promisify from 'pify'
```

Comment:
just using a 3rd party here for correctness and maintainability?

Reply:
Ah I think I can remove this now, was trying this because of some issues with the polyfill